scss/_patterns_lists.scss

```diff
@@ -199,6 +220,84 @@ $list-step-bullet-margin: $sph--x-large;
   }
 }
 
+// Sets horizontal list section grid column count and makes list item borders span the full grid width.
+// $columns: Number of columns that the list items should be divided into.
+// $skip-cols: Number of columns at the start of each row to skip
+@mixin list-horizontal-section-row-width($columns, $skip-cols: 0) {
+  --columns: #{$columns + $skip-cols};
```

Comment:
I really like how this ended up. It looks a bit convoluted in the code, and requires a bit of understanding. But works very effectively and is easy to debug in browser as well.